lib/edit/a_info.txt

```diff
@@ -4517,3 +4517,70 @@ D:それはいつかなされるだろう，しかし，その前にお前がビ
 D:イェンダーの魔除けを取り戻すことができれば，我々は永遠に安泰だ．
 D:「ダイアナの長弓を持って行くがいい．魔除けを探す冒険の助けとなるだろう．」
 D:(勇者オリオン，JNethack)
+
+N:271:ジャック・オーの
+E:of Jack-o
+I:39:4:1
+W:3:1:10:3000
+P:0:1d1:0:0:0
+F:ACTIVATE
+U:DISP_EVIL
+D:$A fancy lantern in the shape of a pumpkin.
+D:$It has a mysterious power to attract good spirits and keep evil spirits away.
+D:カボチャの形をしたファンシーなランタンだ。
+D:善霊を引き寄せ、悪霊たちを遠ざける不思議な力を持っている。
+
+N:272:『高漸離の筑』
+E:'Ancient harp of Gao Jianli'
+I:19:51:1
+W:60:150:200:60000
+P:0:0d0:0:0:0
+F:WIS | CON
+F:ACTIVATE | TELEPATHY | THROW | RES_FEAR | INSTA_ART
+U:SLEEP
+D:$It's a koto-like instrument, but it's strangely heavy.
+D:$For some reason, lead is charged in the body.
+D:$Of course, it's fun to play.
+D:琴のような楽器だが妙に重い。
```

Comment:
For the English description of the ancient harp, I'd use (drawing from https://en.wikipedia.org/wiki/Gao_Jianli):

`D:$It's a koto-like instrument, but it's strangely heavy - with pieces of
 D:$ lead hidden in the body for some reason.
 D:$Of course, it's fun to play.`